common/cluster/metadata.go

```diff
@@ -104,6 +104,8 @@ type (
 		InitialFailoverVersion int64 `yaml:"initialFailoverVersion"`
 		// Address indicate the remote service address(Host:Port). Host can be DNS name.
 		RPCAddress string `yaml:"rpcAddress"`
+		// Cluster ID allows to explicitly set the ID of the cluster. Optional.
+		ClusterID  string `yaml:"-"`
```

Comment:
will it load it from config? if yes, use `yaml:"clusterId"`

Reply:
it's intentionally ignored https://github.com/temporalio/temporal/pull/3883#discussion_r1093896936

This is used by Temporal single binary